modules/learn/pages/buckets-memory-and-storage/storage-settings.adoc

```diff
@@ -95,32 +98,40 @@ For more information, see xref:data/expiration.adoc#post-expiration-purging[Post
+Note that the paths are node-specific:
+consequently, the data for any of these services may occupy a different filesystem-location, on each node.
 
 For information on setting data-paths, see xref:manage:manage-nodes/initialize-node.adoc[Initialize a Node].
 
 [#append-only-writes-and-auto-compaction]
 == Append-Only Writes and Auto-Compaction
 
-Couchbase Server uses an _append-only_ file-write format; which helps to ensure files' internal consistency, and reduces the risk of corruption.
-Necessarily, this means that every change made to a file &#8212; whether an addition, a modification, or a deletion &#8212; results in a new entry being created at the end of the file: therefore, a file whose user-data is diminished by deletion actually grows in size.
+Couchbase Server uses an _append-only_ file-write format,
+which helps to ensure the internal consistency of the files and reduces the risk of corruption.
+Necessarily, this means that every change made to a file &#8212; whether an addition, a modification,
+or a deletion &#8212; results in a new entry being created at the end of the file:
+therefore, a file whose user-data is diminished by deletion actually grows in size.
 
 File-sizes should be periodically reduced by means of _compaction_.
-This operation can be performed either manually, on a specified bucket; or on an automated, scheduled basis, either for specified buckets, or for all buckets.
+This operation can be performed either manually, on a specified bucket; or on an automated, scheduled basis, either for specified buckets or for all buckets.
 
 For information on performing manual compaction with the CLI, see xref:cli:cbcli/couchbase-cli-bucket-compact.adoc[bucket-compact].
 For information on configuring auto-compaction with the CLI, see xref:cli:cbcli/couchbase-cli-setting-compaction.adoc[setting-compaction].
 
-For all information on using the REST API for compaction, see the xref:rest-api:compaction-rest-api.adoc[Compaction API].
+For all information on using the REST API for compaction, see the xref:rest-api:rest-autocompact-global.adoc[Global Compaction API] 
+or xref:rest-api:rest-autocompact-per-bucket.adoc[Per-bucket Compaction API].
 
 For information on configuring auto-compaction with Couchbase Web Console, see xref:manage:manage-settings/configure-compact-settings.adoc[Auto-Compaction].
 
 
 == Disk I/O Priority
 
 _Disk I/O_ — reading items from and writing them to disk — does not block client-interactions: disk I/O is thus considered a _background task_.
-The priority of disk I/O (along with that of other background tasks, such as item-paging and DCP stream-processing) is configurable _per bucket_.
-This means, for example, that one bucket's disk I/O can be granted priority over another's.
+The priority of disk I/ O, (along with that of other background tasks, such as item-paging and DCP stream-processing) is configurable _per bucket_.
```

Comment:
Has there been a space added here?

Reply:
Mmm 🤔 

It looks like I've added a comma, which now doesn't look right. I'll remove it.